src/structures/VoiceChannel.js

```diff
@@ -8,22 +8,13 @@ const Permissions = require('../util/Permissions');
  * @extends {BaseGuildVoiceChannel}
  */
 class VoiceChannel extends BaseGuildVoiceChannel {
-  /**
-   * Whether the channel is deletable by the client user
-   * @type {boolean}
-   * @readonly
-   */
-  get deletable() {
-    return super.deletable && this.permissionsFor(this.client.user).has(Permissions.FLAGS.CONNECT, false);
-  }
-
   /**
    * Whether the channel is editable by the client user
    * @type {boolean}
    * @readonly
    */
   get editable() {
```

Comment:
Probably out of scope for this PR, but we should deprecate this getter.
Having both `VoiceChannel#editable` and `VoiceChannel#manageable` is plain confusing.